contracts/governance/TimelockController.sol

```diff
@@ -67,14 +67,15 @@ contract TimelockController is AccessControl, IERC721Receiver, IERC1155Receiver
      * proposer and the canceller role (for backward compatibility). The
      * executors receive the executor role.
      *
-     * NOTE: At construction, both the deployer and the timelock itself are
-     * administrators. This helps further configuration of the timelock by the
-     * deployer. After configuration is done, it is recommended that the
-     * deployer renounces its admin position and relies on timelocked
-     * operations to perform future maintenance.
+     * NOTE: At construction, an optional admin can be set (in addition to the
+     * timelock itself). Setting such an admin can help further configuration
+     * of the timelock. After configuration is done, it is recommended to have
+     * this admin renounces its role and to rely on timelocked operations to
+     * perform future maintenance.
      */
     constructor(
         uint256 minDelay,
+        address admin,
```

Comment:
I would add the argument at the end, it makes the breaking change somewhat more gentle, I think...

Reply:
Agree. End is better.

Reply:
done